packages/frontend/app/(dashboard)/course/[cid]/(settings)/settings/queue_invites/components/QueueInvite.tsx

```diff
@@ -0,0 +1,237 @@
+'use client'
+
+import {
+  Button,
+  Checkbox,
+  Form,
+  Input,
+  List,
+  Popconfirm,
+  Select,
+  message,
+} from 'antd'
+import type { QueueInvite, QueueInviteParams } from '@koh/common'
+import { getErrorMessage } from '@/app/utils/generalUtils'
+import { API } from '@/app/api'
+import { CopyOutlined, DeleteOutlined } from '@ant-design/icons'
+import Link from 'next/link'
+import { useEffect, useState } from 'react'
+import _ from 'lodash'
```

Comment:
lodash was causing Dima some issues in an old PR. This works fine for me, but maybe try manually testing creating a queue invite and editing it.